src/components/Controllers/ChefController.jsx

```diff
@@ -1,28 +1,39 @@
-class ChefController {
-  static create() {
-    const name = window.prompt('Enter the name for the new Chef:');
-    if (!name) return null;
-
-    const image = window.prompt('Enter the image URL for the new Chef:');
-    if (!image) return null;
+import React from 'react';
+import UploadWidget from '../Dashboard/EditForm/UploadWidget';
 
-    const description = window.prompt('Enter the description for the new Chef:');
-    if (!description) return null;
+class ChefController {
 
-    return {
-      name,
-      image,
-      description,
+  static renderCreateForm({ newItem, handleChange }) {
+    const handleImageUploadSuccess = (imageUrl) => {
+      handleChange({ target: { name: 'image', value: imageUrl } });
     };
+
+    return (
+      <>
+        <label>Name:</label>
+        <input type="text" name="name" value={newItem.name} onChange={handleChange} />
+        <label>Image:</label>
+        <UploadWidget onSuccess={handleImageUploadSuccess} />
+        <label>Description:</label>
+        <input type="text" name="description" value={newItem.description} onChange={handleChange} />
+        <label>Restaurants:</label>
+        <input type="text" name="restaurants" value={newItem.restaurants} onChange={handleChange} />
+      </>
```

Comment:
use resources file and use it in map here as 
```suggestion

const chefFields = [{
label: RESOURCES.NAME, 
component:  <input type="text" name="name" value={newItem.name} onChange={handleChange} />
}]
      <>
      {chefFields.map((chefField)=><>
      <label>chefLebel </label>
      {chefField.component}
      </>}
  
      </>
```